moveit_ros/perception/mesh_filter/src/depth_self_filter_nodelet.cpp

```diff
@@ -206,8 +206,8 @@ void mesh_filter::DepthSelfFiltering::connectCb()
   else if (!sub_depth_image_)
   {
     image_transport::TransportHints hints("raw", ros::TransportHints(), getPrivateNodeHandle());
-    sub_depth_image_ =
-        input_depth_transport_->subscribeCamera("depth", queue_size_, &DepthSelfFiltering::depthCb, this, hints);
+    sub_depth_image_ = input_depth_transport_->subscribeCamera("depth", queue_size_, &DepthSelfFiltering::depthCb, this,
```

Comment:
`queue_size_` defaults to `1`, the profile `rmw_qos_profile_sensor_data` has a default history size of 5 which could be customized. I'm not sure if there are any best practices for matching these values.

Reply:
I also don't know which one takes precedence. To err on the side of caution, modify the profile history depth to match whatever value the `queue_` variable has?